lib/components/user/stacked-pane-display.js

```diff
@@ -10,8 +10,12 @@ import { PageHeading, StackedPaneContainer } from './styled'
  */
 const StackedPaneDisplay = ({ onCancel, paneSequence, title }) => (
   <>
-    <BackToTripPlanner />
-    {title && <PageHeading>{title}</PageHeading>}
+    {title && (
+      <PageHeading>
+        <Button bsStyle='primary' className='pull-right' type='submit'>Save</Button>
```

Comment:
I'm not a huge fan of the Save button up here (at least for now). I think we want the user to take the action of scrolling through the form to view/confirm the settings before saving.

Reply:
That was an experiment... It's reverted by https://github.com/opentripplanner/otp-react-redux/pull/323/commits/8dc59719a85afdc13f901d439d9304a701c47e9c.